database/dbHelper.js

```diff
@@ -0,0 +1,23 @@
+const db = require("./dbConfig");
+
+module.exports = {
+  register: register,
+  getUsers: getUsers,
+  login: login
+};
+
+function register(newUser) {
+  return db("users").insert(newUser);
+}
+
+function login(username) {
+  console.log(username);
+  return db("users")
+    .where({ username: username })
```

Comment:
i know you like to be explicit, but just a reminder, you can shorten duplicate key:value pairs in es6 to a single value...    .where({username}) will do the same thing